### ISSUE TITLE
no-nested-template-literals no need to report an error if there are no placeholders in the nested literal

### ISSUE THREAD

**Issue by WiseBird** [SonarSource/eslint-plugin-sonarjs#292]
**I want to report a bug.**

**Reproducer**

```javascript
console.log(`Look at ${red(`can't`)} do this`); // gives an error
console.log(`I ${red("can't")} do this`); // doesn't
```

**Expected behavior**

Rule shouldn't report an error if there are no placeholders in the nested literal.

**eslint-plugin-sonarjs version:** 0.10.0<br>
**eslint version:** 8.1.0<br>
**Node.js version:** v16.5.0<br>

**Rule key:** no-nested-template-literals

**Comment by vilchik-elena** [comment]
IMO even if there is no placeholders it's still relevant to refactor, especially as you can simply use quotes.

**Comment by Wohops** [comment]
This issue has been migrated to Jira. [ESLINTJS-16](https://sonarsource.atlassian.net/browse/ESLINTJS-16)
